libutils/mustache.c

```diff
@@ -490,6 +490,13 @@ static bool RenderVariable(Buffer *out,
             // because we return false earlier otherwise
             return RenderVariablePrimitive(out, var, escape, json_key);
         }
+        else if (item_mode)
+        {
+            Log(LOG_LEVEL_WARNING,
+                "{{.}} mustache tag cannot expand a container without specifying conversion"
+                " - consider {{$.}} or {{%%.}}");
```

Comment:
Shouldn't we do the same thing as https://mustache.github.io/#demo does in such cases?

Reply:
See discussion in JIRA:
https://tracker.mender.io/browse/CFE-3457
https://tracker.mender.io/browse/CFE-3489

Especially this comment:
https://tracker.mender.io/browse/CFE-3457?focusedCommentId=110897&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-110897

Reply:
@vpodzime I added a comment to explain and information and links to the commit message.

Reply:
> @vpodzime I added a comment to explain and information and links to the commit message.

Good enough for me! :+1: Thanks!